app/Question.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Question extends Model
+{
+	/**
+	*	Attributes to be mass filled.
+	*/
+    protected $fillable = [
+        'user_id', 'organization_id', 'event_id', 'question', 
+        'question_body', 'answer', 'answered_at'
+    ];
+    /**
```

Comment:
1.ids will not be mass assigned. Typically, we will create a new question and according to the eloquent relationships, it will be saved in a given user questions and in a given event questions. 
2. We do not need to have a body for the question. One field for the question will be enough.